pkg/block/block.go

```diff
@@ -153,3 +153,10 @@ func IsBlockDir(path string) (id ulid.ULID, ok bool) {
 	id, err := ulid.Parse(filepath.Base(path))
 	return id, err == nil
 }
+
+func HasMetaFile(blockPath string) bool {
```

Comment:
I am against this shallow function - such helper is not needed - the oneliner below is exactly enough for this.

In the same way, there is no `Max(a, b)` function is go standard library. You just do this one `if ` on your own (: 
What do you think, can we just inline this logic?

Reply:
We could do what @bwplotka suggests, it might be neater. :+1:

Reply:
I eventually simplified it to checking directly for the meta file which should also provide the check for the directory existence right away. The creation of dir is agnostic to it's existence and new downloaded files should overwrite those existing in the block dir. 

Thanks @bwplotka for pointing this out I just saw the `IsBlockDir` func so I somehow followed the pattern.